docs/docs/target/kubernetes.md

```diff
@@ -33,55 +33,45 @@ Trivy follows the behavior of the `kubectl` tool as much as possible.
 ### Scope
 
 The command expects an argument that selects the scope of the scan (similarly to how `kubectl` expects an argument after `kubectl get`). This argument can be:
-1. A Kubernetes Kind. e.g `pod`, `deployment`, etc. 
+
+1. A Kubernetes Kind. e.g `pod`, `deployment`, etc.
 2. A Kubernetes Resource. e.g `pods/mypod`, etc.
-3. `all`. Scan common workload kinds, as listed [here](https://github.com/aquasecurity/trivy-kubernetes/blob/bf8cc2a00d9772e0aa271f06d375b936152b54b1/pkg/k8s/k8s.go#L296:L314)
-4. `cluster` scan the entire cluster including all namespaced resources and cluster level resources.
 
 Examples:
 
+```sh
+trivy k8s --report summary
 ```
-trivy k8s all
-trivy k8s pods
-trivy k8s deploy myapp
-trivy k8s pod/mypod
-trivy k8s pods,deploy
-trivy k8s cluster
-```
-
-Note that the scope argument must appear last in the command line, after any other flag.
-
-### Cluster
 
 By default Trivy will look for a [`kubeconfig` configuration file in the default location](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/), and use the default cluster that is specified.  
 You can also specify a `kubeconfig` using the `--kubeconfig` flag:
 
-```
+```sh
 trivy k8s --kubeconfig ~/.kube/config2
 ```
 
 ### Namespace
```

Comment:
didn't this flag change?

Reply:
you mean `--context` flag ?

Reply:
no, I mean we no londer have a --namespace flag no?